tests/Agent/IntegrationTests/ContainerApplications/docker-compose-memcached.yml

```diff
@@ -0,0 +1,56 @@
+# The following must be set either in environment variables or via a .env file in the same folder as this file:
+#
+# AGENT_PATH      host path to the Agent linux home folder - will map to /usr/local/newrelic-dotnet-agent in the container
+# LOG_PATH        host path for Agent logfile output - will map to /app/logs in the container
+# DISTRO_TAG      distro tag for build, not including the architecture suffix - possible values 8.0-bookworm-slim, 8.0-alpine, 8.0-jammy
+# TARGET_ARCH     the target architecture for the build and run -- either amd64 or arm64
+# PORT            external port for the smoketest API
+# CONTAINER_NAME  The name for the container
+# PLATFORM        The platform that the service runs on -- linux/amd64 or linux/arm64/v8
+# DOTNET_VERSION  The dotnet version number to use (8.0, etc)
+# TEST_DOCKERFILE The path and dockerfile to use for the service.
+#
+# NEW_RELIC_KAFKA_TOPIC
+# 
+# and the usual suspects:
+# NEW_RELIC_LICENSE_KEY
+# NEW_RELIC_HOST
+# NEW_RELIC_APP_NAME
+#
+#
+# To build and run, execute `docker compose -f <path to docker-compose.yml> up` 
+# Alternatively, set COMPOSE_FILE environment variable to the path and omit the -f parameter
+
+services:
+    LinuxSmokeTestApp:
```

Comment:
Should this name be changed to reflect that this is the membcachd test app?

Reply:
I used the kafka yml as the base for this and it also has the generic test name.   I can change it.